### ISSUE TITLE
Automatic Presentation Context Broken

### ISSUE THREAD

**Issue by jadar** [SwiftKickMobile/SwiftMessages#200]
I'm playing around with your demo app, and it seems like the "automatic" presentation context is broken. I would like to use your library and I'd like to see how these configurations work. Any ideas?

Thanks.

**Comment by wtmoose** [comment]
Could you elaborate on what you mean by broken?

**Comment by jadar** [comment]
I apologize. I mean it simply doesn't show a message.
![2018-07-24 17 28 57](https://user-images.githubusercontent.com/3793928/43167364-644fe53a-8f67-11e8-8713-d0c0a2201915.gif)

iOS 11.4
Xcode 9.4.1

**Comment by jadar** [comment]
I did some debugging and it seems that the demo app is either getting compiled with the `SWIFTMESSAGES_APP_EXTENSIONS` flag, or it's using the wrong binary. Whether the Messages app is running or not, the `throw SwiftMessagesError.noRootViewController` code path  in `Presenter.getPresentationContext` is always taken.

If it helps, the previous commit (before the merging of #191) seems to work properly.

Hope this helps.

**Comment by wtmoose** [comment]
Yep, I was just looking into it. Somehow the demo project is getting linked to two different SwiftMessages framework builds (one for apps and one for iMessage extensions) and the wrong one is being selected. Thanks for bringing this up.
